### PR TITLE
chore: fix typos with codespell

### PR DESCRIPTION
**Pull request by Jolg42**
I wanted to try [codespell](https://github.com/codespell-project/codespell) and thought it would be a good idea here :)

### CODE REVIEW

File: packages/cli/src/lib/pluralHelpers.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const ensureUniquePlural = async ({
 
   const promptMessage = isDestroyer ? destroyMessage : generateMessage
 
-  // News => Newses; Equipment => Equipments
+  // News => Newses; Equipment => Equipment
```
Comment (Jolg42) [inline packages/cli/src/lib/pluralHelpers.js:47]:
```suggestion
  // News => Newses; Equipment => Equipments
```

Comment (Jolg42) [reply packages/cli/src/lib/pluralHelpers.js:47]: Reverted ✅

**Review by Jolg42 — COMMENTED** [review]
I just double (or maybe triple checked by now 😄) checked all changes and everything looks good to me ✨

**Review by thedavidprice — APPROVED** [review]
Wow, this works really well! I'm approving the changes but making sure we get final approval from @jtoar because I want to make sure this is correclty handling previous versions of docs.

- [ ] this updates snapshots and tests; need to run CI
- [ ] this updates previous versions of docs; need to build+deploy but not trigger Algolia
- [ ] Get Dom's "Ok" to merge

**Comment by Jolg42** [comment]
@thedavidprice It works well but I still had to make sure to not commit the bad changes, mainly many `afterAll` that it somehow wanted to change to `after all` 🙈 😄

**Comment by Jolg42** [comment]
I knew CSpell from https://github.com/typescript-eslint/typescript-eslint/blob/main/.cspell.json and others and the VS Code extension is good but there is no "apply all suggested fixes" feature there that I could find.

File: packages/router/src/__tests__/router.test.tsx [hunk 2]
```diff
@@ -1616,7 +1616,7 @@ describe('Unauthorized redirect error messages', () => {
     console.error = jest.fn()
   })
 
-  afterAll(() => {
+  after all(() => {
```
Comment (Jolg42) [inline packages/router/src/__tests__/router.test.tsx:1619]: Revert
```suggestion
  afterAll(() => {
```

**Comment by Jolg42** [comment]
(done now) ✅

**Review by jtoar — APPROVED** [review]
Wow we had a lot more of these than I ever would've thought. Thanks @Jolg42!

**Comment by Jolg42** [comment]
You're welcome!
Funnily it's spread on many files, but my guess is that given the number of lines you have in the repo it's not a lot 😄

**Pull request merged**